src/app/helpers/hero-xp.ts

```diff
@@ -33,6 +33,12 @@ export function heroLevelUp(hero: Hero): void {
   if (newHero) {
     recalculateStats(newHero.id);
   }
+
+  // Clear the "too hard" nodes list when any hero levels up
+  updateGamestate((state) => {
+    state.hero.tooHardNodes = [];
+    return state;
+  });
```

Comment:
refactor this to a function in `world.ts` called `clearNodesTooHardForHeroes` rather than updating it here.